cli/lockfile.rs

```diff
@@ -42,13 +45,25 @@ impl Lockfile {
     }
     let j = json!(&self.map);
     let s = serde_json::to_string_pretty(&j).unwrap();
+    let s_options = FmtOptionsConfig {
+      use_tabs: Some(false),
+      line_width: Some(80),
+      indent_width: Some(2),
+      single_quote: Some(true),
+      prose_wrap: Some(ProseWrap::Preserve),
+    };
+    let formated_s = format_json(&s, &s_options);
```

Comment:
Instead of `s_options` you should be able to do `&Default::default()` here. That will get it to use the default config and then we don't need to have the default values stored in multiple places.